Pollo/ViewControllers/PollsViewController.swift

```diff
@@ -323,34 +381,9 @@ class PollsViewController: UIViewController {
         }
     }
     
-    func displayNewGroupActivityIndicatorView() {
-        newGroupButton.isHidden = true
-        newGroupButton.isUserInteractionEnabled = false
-        
-        newGroupActivityIndicatorView.isHidden = false
-        newGroupActivityIndicatorView.isUserInteractionEnabled = true
-        newGroupActivityIndicatorView.startAnimating()
-    }
-    
-    func hideNewGroupActivityIndicatorView() {
-        newGroupActivityIndicatorView.stopAnimating()
-        newGroupActivityIndicatorView.isHidden = true
-        newGroupActivityIndicatorView.isUserInteractionEnabled = false
-        
-        newGroupButton.isHidden = false
-        newGroupButton.isUserInteractionEnabled = true
-    }
-    
-    func joinSessionWithCode(with code: String) -> Future<Data> {
-        return networking(Endpoint.joinSessionWithCode(with: code))
-    }
-    
-    func getSortedPolls(with id: Int) -> Future<Data> {
-        return networking(Endpoint.getSortedPolls(with: id))
-    }
-    
     @objc func joinSession() {
         guard let code = codeTextField.text, code != "" else { return }
+        codeTextField.text = ""
```

Comment:
are we sure that we want to clear the textfield right away? I feel like it might be odd if we clear the textfield and the network request takes a while to respond.
as a side note, is there some loading indicator displayed when they click join?

Reply:
I will change this so that it is cleared later in the function once the request is successful. As for your followup question, there is no loading indicator, the request is very quick almost instantaneous.

Reply:
But if your network connection is slow, we would want to have some sort of user feedback so they know that their group is being created in case the network request doesn't return right away. I'd suggest adding a loading indicator, maybe over the "Create" button?